panel.go

```diff
@@ -68,6 +69,10 @@ type Panel struct {
 	TooltipImmediate     bool
 }
 
+func (p *Panel) SetParent(panel *Panel) {
```

Comment:
As I think I've stated before, I will not add a `.SetParent(*Panel)` method. If you want to set the parent, call `.AddChild(Paneler)` or `.AddChildAtIndex(Paneler,int)` on the parent instead. This will ensure all of the other things that clients expect to happen, do happen, unlike with what this method is doing.